src/sonic-frr/patch/0054-Updating-rules-to-set-multipath-to-512-and-removing-.patch

```diff
@@ -0,0 +1,23 @@
+From ca51d39440db46e766a81987f407ef794d697876 Mon Sep 17 00:00:00 2001
+From: dgsudharsan <sudharsand@nvidia.com>
+Date: Fri, 8 Nov 2024 22:14:31 +0000
+Subject: [PATCH] Updating rules to set multipath to 512 and removing bgp vnc
+
+
+diff --git a/debian/rules b/debian/rules
+index aa13106fec..e280b6c45b 100755
+--- a/debian/rules
++++ b/debian/rules
+@@ -68,8 +68,7 @@ override_dh_auto_configure:
+ 		--disable-protobuf \
+ 		--disable-zeromq \
+ 		--enable-ospfapi \
+-		--enable-bgp-vnc \
+-		--enable-multipath=256 \
```

Comment:
what is the use case to have 512 ECMP paths from FRR. 

Currently FRR uses uint8 for num paths.

Reply:
This is fixed by FRR https://github.com/FRRouting/frr/pull/16967. We will be taking this fix once FRR upgrade is done. We do have some specific use case scenarios where the number of nexthops might scale more than 256. So we are setting to 512 here.

Reply:
You can refer to the topology here https://github.com/sonic-net/sonic-mgmt/pull/15355